src/tutorials/0006/04.md

```diff
@@ -0,0 +1,15 @@
+So now our Version 1 CID contains the following fields:
+
+`<multicodec><multihash>`
```

Comment:
You might want to start with multihash expanded out as `<mh-algo><mh-len><mh-hash>` before shortening to `<multihash>` as a reminder.

Reply:
```suggestion
`<ipld-format><multihash>`
```